src/components/app/MenuButtons/index.js

```diff
@@ -205,8 +205,8 @@ class MenuButtonsImpl extends React.PureComponent<Props> {
   render() {
     return (
       <>
-        {this._renderMetaInfoButton()}
         {this._renderRevertProfile()}
+        {this._renderMetaInfoButton()}
```

Comment:
Surprisingly, we have no test for this... I did try to write one but kept running into weird problems with jest fake timers and react-testing-library that I don't really want to debug now :/